internal/proxy/multi_app_conn.go

```diff
@@ -33,7 +33,7 @@ type AppConns interface {
 }
 
 // NewAppConns calls NewMultiAppConn.
-func NewAppConns(clientCreator ClientCreator) AppConns {
+func NewAppConns(clientCreator abciclient.Creator) AppConns {
```

Comment:
Seems like we could drop one of these two constructor? Not necessary for this change, but I'm not sure we need both.

Reply:
probably? but it's internal now, and I would prefer if this patch just had moves/renames and not anything more.